src/docfx/config/ConfigLoader.cs

```diff
@@ -111,14 +111,15 @@ private static JObject LoadConfig(List<Error> errorBuilder, string fileName, str
         }
 
         private JObject DownloadExtendConfig(
-            List<Error> errors, string locale, PreloadConfig config, Repository repository, FileResolver fileResolver)
+            List<Error> errors, PreloadConfig config, JObject opsConfig, Repository repository, FileResolver fileResolver)
```

Comment:
Prefer strong class here. Can make `LoadDocfxConfig` return `(docfxConfig, opsConfig)`

```suggestion
            List<Error> errors, PreloadConfig config, OpsConfig opsConfig, Repository repository, FileResolver fileResolver)
```